.github/workflows/draft-pdf.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [main, master]
+  pull_request:
```

Comment:
Does this mean this draft will get rebuilt on every push/pull request? Might be better to just do this on merge to `main`?

Reply:
(PS in general, do we need to include `master` here? since we only really use `main`...)

Reply:
I had just copied this file from JOSS. But we can get rid of `master` here without any issues.

We can delete this workflow once the paper is published. I have added a TODO to that effect.